generated/python/gapic-google-cloud-pubsub-v1/google/cloud/gapic/pubsub/v1/publisher_client.py

```diff
@@ -213,8 +204,7 @@ def __init__(self,
             client_config,
             config.STATUS_CODE_NAMES,
             metrics_headers=metrics_headers,
-            bundle_descriptors=self._BUNDLE_DESCRIPTORS,
```

Comment:
That's convincing enough to me, especially since SemVer technically doesn't make any such guarantees about breaking patch changes in version 0.